Shaders/compile.sh

```diff
@@ -1,26 +1,26 @@
 #!/bin/bash
 
 #set your VULKAN_SDK location  before running
-VULKAN_SDK=~/VulkanSDK/1.0.26.0/x86_64/
+#VULKAN_SDK=~/VulkanSDK/1.0.26.0/x86_64/
```

Comment:
Instead of commenting this out, you can conditionalize it by adding an environment check around it

Reply:
Sure, we could do that. But this should be instead done / handled by default by the Vulkan SDK (and the user should follow their instructions into how to install it and setup their Vulkan SDK)

As per the official Vulkan SDK, they provide a very neat and clean way for this. They recommend adding the following to your "~/.bash_profile"

`# Vulkan SDK Stuff
source /opt/vulkan-sdk/USING/setup-env.sh
`